python/common/org/python/types/Int.java

```diff
@@ -78,7 +78,15 @@ public Int(org.python.Object[] args, java.util.Map<java.lang.String, org.python.
                     );
                 }
             }
-        } else if (args.length > 3) {
+        } else if (args.length == 2) {
+            try {
+                org.python.types.Str argStr = (org.python.types.Str) args[0];
+                org.python.types.Int argInt = (org.python.types.Int) args[1];
+                this.value = java.lang.Integer.parseInt(argStr.value, (int) argInt.value);
+            } catch (Exception e) {
```

Comment:
This should probably be catching only `NumberFormatException`, it's better to catch specific exceptions than use a catch-all with `Exception`.